integrations/kubernetes-ingress/external-secrets.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: ngrok External Secrets integration
+description: ngrok integrates with External Secrets Operator to enable secure management of secrets used in ngrok Traffic Policy.
+sidebarTitle: External Secrets
+---
+
+This integration allows you to push Kubernetes secrets to ngrok vaults, where they can be referenced in your traffic policies for secure configuration management.
+
+## How it works
+
+[External Secrets Operator](https://external-secrets.io/) watches for PushSecret resources in your cluster.
+When a PushSecret is created or updated, it reads the specified Kubernetes secret and pushes the secret data to your ngrok vault using the ngrok API.
+The secret then becomes available in ngrok for use in Traffic Policies.
+The operator continues to sync changes based on the configured refresh interval, ensuring your ngrok secrets stay up-to-date with changes in Kubernetes.
+
+### Limitations
+
+- Currently supports push operations only (Kubernetes → ngrok)
```

Comment:
```suggestion
- Currently supports push operations only (External Secret Store → ngrok)
```

@jonstacks Do we want to say Kubernetes → ngrok? Or something more... generic?

Reply:
I think its fine. Maybe `Kubernetes Secrets` if we are really wanting to change it.